src/Dialogs/ProfileEdit.vala

```diff
@@ -415,7 +415,7 @@ public class Tuba.Dialogs.ProfileEdit : Adw.Window {
 
 			chooser.response.connect (id => {
 				switch (id) {
-					case ResponseType.ACCEPT:
+					case Gtk.ResponseType.ACCEPT:
```

Comment:
No idea why it was working before and not working now to omit `Gtk.` here

Reply:
My bad! Copy-pasted the GTK < 4.10 deprecation compat without checking it, unsure why the < 4.10 was used considering adw 1.4 requires 4.11 :/ (either way it will be removed on when adw 1.4)